air/src/execution_step/boxed_value/stream.rs

```diff
@@ -208,9 +210,13 @@ impl Stream {
     }
 
     /// Removes empty generations from current values.
-    fn remove_empty_generations(&mut self) {
+    pub(crate) fn remove_empty_generations(&mut self) {
         self.values.retain(|values| !values.is_empty());
     }
+
+    pub(crate) fn get_values_iter(&self) -> Iter<'_, Vec<ValueAggregate>> {
+        self.values.iter()
+    }
```

Comment:
Thanks to reusing `Display` for stream you don't need this function.

Reply:
Done